packages/pyml/pyml.20190626/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://github.com/thierry-martinez/pyml"
+bug-reports: "http://github.com/thierry-martinez/pyml/issues"
+license: "BSD"
+dev-repo: "git+https://github.com/thierry-martinez/pyml.git"
+build: [make "all" "pymltop" "pymlutop" {utop:installed} "PREFIX=%{prefix}%"]
+install: [make "install" "PREFIX=%{prefix}%"]
+remove: [make "uninstall" "PREFIX=%{prefix}%"]
+synopsis: "OCaml bindings for Python"
+description: "OCaml bindings for Python 2 and Python 3"
+depends: [
+  "ocaml" {>= "3.12.1" & < "4.09.0"}
+  "ocamlfind" {build}
+  "stdcompat" {>= "9"}
+  "num"
+]
+depopts: ["utop"]
+version: "20190626"
```

Comment:
```suggestion
```

This is redundant, opam will infer the version from the package folder name and override it

Reply:
Thank you! Fixed.